pkg/security/podsecuritypolicy/capabilities/capabilities.go

```diff
@@ -36,11 +37,23 @@ var _ Strategy = &defaultCapabilities{}
 
 // NewDefaultCapabilities creates a new defaultCapabilities strategy that will provide defaults and validation
 // based on the configured initial caps and allowed caps.
-func NewDefaultCapabilities(defaultAddCapabilities, requiredDropCapabilities, allowedCaps []api.Capability) (Strategy, error) {
+func NewDefaultCapabilities(defaultAddCapabilities, requiredDropCapabilities, allowedCaps []corev1.Capability) (Strategy, error) {
+	internalDefaultAddCaps := []api.Capability{}
```

Comment:
initialize these to the length of the slice you're copying in to avoid reallocation while appending